### DISCUSSION TITLE
problem adding records to queue in crowdsourcing

### DISCUSSION THREAD

**Discussion by mgoliver22** [Symbiota/community-board#402] [Problems/Bugs]
Hello! I've been checking how many records I can add to the crowdsourcing queue for the collection I'm associated with (TENN) on the Bryophyte Portal in anticipation for WeDigBio next week. 
When I go into Processing Toolbox and select Crowdsourcing, it says there are 635 records available to add. After I add all of them to the queue it ends up only being 213. If I do a search of all unprocessed specimens that have images associated with them I do get back 213 records. Where does this big number discrepancy come from and are there are a bunch of other records somewhere I could be adding but can't for some reason? Or are there really only 213 records but they have 635 images associated with them and that's where that number is coming from? 
If I do a search for all unprocessed specimens regardless of whether they have images or not I currently get back 267 records so it's not as simple as there being 422 unprocessed records that don't have images yet. I also checked to see whether there are a bunch of records without a processing status in case that was somehow blowing up the "available to add" to crowdsourcing queue number, but that also doesn't seem to be the issue.

Records available to add to the queue, 635:
![available to add to queue](https://github.com/BioKIC/symbiota-docs/assets/66650858/d95c2dd7-fb8a-498a-85da-6015ffbf0c56)

Records after I've added them all to the queue, 213:
![all added to queue](https://github.com/BioKIC/symbiota-docs/assets/66650858/73abffc9-abc5-461d-9382-8edb0ce89565)

**Comment by themerekat** [comment]
(responded also via email)

I think I have identified the problem. The query to determine the number of records to add actually gives you the number of ﻿images﻿ associated with records that you could add to the crowdsourcing queue, rather than the number of records itself. We will make a note to amend this in the code, as it is misleading.

**Reply by mgoliver22** [reply]
Thanks, Katie!
